kofta/public/locales/fr/translation.json

```
@@ -39,7 +39,7 @@
     "home": {
       "createRoom": "Créer une salle",
       "refresh": "Actualiser",
-      "editRoom": "Edit Room"
+      "editRoom": "Modifier la salle "
```

Why is there an extra space?
```suggestion
      "editRoom": "Modifier la salle"
```